ci-operator/config/openshift/special-resource-operator/openshift-special-resource-operator-master.yaml

```diff
@@ -36,19 +40,120 @@ releases:
 resources:
   '*':
     limits:
-      memory: 4Gi
+      memory: 8Gi
     requests:
-      cpu: 100m
-      memory: 200Mi
+      cpu: 300m
+      memory: 500Mi
 tests:
+- as: lint
+  commands: |
+    export XDG_CACHE_HOME=/tmp/.cache
+    make lint
+  container:
+    clone: true
+    from: sro-test-bin
+- as: helm-lint
+  commands: make helm-lint
+  container:
+    clone: true
+    from: sro-test-bin
+- as: kube-lint
+  commands: make kube-lint
+  container:
+    from: src
 - as: unit
-  commands: make test
+  commands: make unit-test
   container:
     from: src
 - as: verify
   commands: make verify
   container:
     from: src
+- as: e2e-aws-ocp
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.9"
+  steps:
+    test:
+    - as: deploy-nfd
+      cli: latest
+      commands: |
+        git clone --branch release-4.9 https://github.com/openshift/cluster-nfd-operator
+        cd cluster-nfd-operator
+        make deploy VERSION=release-4.9
+        oc apply -f config/samples/nfd.openshift.io_v1_nodefeaturediscovery.yaml
+      from: sro-test-bin
+      resources:
+        requests:
+          cpu: 300m
+          memory: 500Mi
+    - as: deploy-sro
+      cli: latest
+      commands: make deploy
+      dependencies:
+      - env: IMG
+        name: special-resource-rhel8-operator
+      from: sro-test-bin
+      resources:
+        requests:
+          cpu: 300m
+          memory: 500Mi
+    - as: test
+      cli: latest
+      commands: make e2e-test
+      from: src
+      resources:
+        requests:
+          cpu: 300m
+          memory: 500Mi
+    workflow: generic-claim
+- as: e2e-aws-ocp-4-8-periodic
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.8"
+  cron: 38 12 * * *
+  steps:
+    test:
+    - as: deploy-nfd
+      cli: latest
+      commands: |
+        git clone --branch release-4.9 https://github.com/openshift/cluster-nfd-operator
```

Comment:
This is still `4.9`. It should be `4.8` - you can see that that this is a specific job for `4.8`, that means that:
1. OCP should run `4.8` as you can see #114 and #121
2. NFD should run `4.8` as well - this is what needs to be changed (currently `4.9`)